catalog/app/utils/Config.js

```diff
@@ -76,6 +76,7 @@ const transformConfig = (cfg) => ({
   s3Proxy: startWithOrigin(cfg.s3Proxy),
   apiGatewayEndpoint: startWithOrigin(cfg.apiGatewayEndpoint),
   binaryApiGatewayEndpoint: startWithOrigin(cfg.binaryApiGatewayEndpoint),
+  desktop: AUTH_MAP[cfg.desktop],
```

Comment:
we should not reuse `AUTH_MAP`, bc it was introduced specifically for auth settings